utils/wm-utils/src/main/java/com/microsoft/device/dualscreen/utils/wm/FoldingFeatureExtensions.kt

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+package com.microsoft.device.dualscreen.utils.wm
+
+import android.graphics.Rect
+import androidx.window.layout.FoldingFeature
+
+/**
+ * On a foldable device will return the hinge position as a [Rect].
+ * If the device is not foldable or it doesn't have a hinge, will return null.
+ */
+fun FoldingFeature.getHinge(): Rect {
+    return bounds
+}
+
+/**
+ * Checks if the application is spanned on both screens.
+ * @return true if is spanned or false if is not spanned or the device doesn't have to displays
+ */
+fun FoldingFeature?.isSpanned(): Boolean {
+    return this?.let { feature ->
+        feature.state == FoldingFeature.State.FLAT && feature.isSeparating
```

Comment:
This will never be true for Type.FOLD, since isSeparating is true only for state == FoldingFeature.State.HALF_OPENED and the state cannot be both FLAT and HALF_OPENED at the same time. Is this intended?

Reply:
This is very important ^^